mzLib/Test/FileReadingTests/TestPsmFromTsv.cs

```diff
@@ -286,29 +287,20 @@ public static void TestParseModification()
             modDict = SpectrumMatchFromTsv.ParseModifications(twoMods.FullSequence);
             Assert.That(modDict.Count == 2);
             Assert.That(modDict.ContainsKey(0) && modDict.ContainsKey(104));
-            Assert.That(modDict[0].Count == 1);
```

Comment:
Why remove this assert and the one below?

Reply:
The dictionary values are no longer iterables (lists of mods) that can be counted. Now, the values are strings (the only mod at the position).